lib/infusionsoft/api.rb

```diff
@@ -221,5 +221,11 @@ def fetch_contacts_with_tag(tag_id, custom_fields = [], page = 0)
       end
       contacts
     end
+
+    def get_contacts(page = 0, custom_fields = [], data={})
```

Comment:
1. We had a method called get_tags, so I thought get_contacts would be ok! but I'm changing it right now!